packages/mdc-drawer/slidable/foundation.js

```diff
@@ -54,7 +54,6 @@ export class MDCSlidableDrawerFoundation extends MDCFoundation {
 
     this.inert_ = false;
 
-    this.drawerClickHandler_ = (evt) => evt.stopPropagation();
```

Comment:
There are still 2 references to `drawerClickHandler_` in this file that should be removed.